book-1-orientation/chapters/QUEST.md

```diff
@@ -0,0 +1,287 @@
+# An Adventurer's Quest
+
+## A C# Exercise
+
+This exercise we will don the brightly colored robe and shiny hat of an adventurer and undergo a series of challenges to complete a quest.
+
+### Phase 1
+
+In this phase we'll get going with some starter code. We'll add to this initial code in future phases.
+
+1. Create a new console application called _"Quest"_.
+1. Initialize a git repo for the new app, add a `.gitignore` file and commit the code.
+1. Create a new repo in GitHub and push your code to it.
+1. Add two new C# source code files. _(Do not remove the `Program.cs` file)_
+    * `Adventurer.cs`
+    * `Challenge.cs`
+1. Update the `*.cs` files to contain the code shown below.
+
+    > `Program.cs`
+
+    ```cs
+    using System;
+    using System.Collections.Generic;
```

Comment:
Is it too much info here to have a comment talking a bit about the namespace, System.Collections.Generic? Just so that they realize in the future that if they want something like a List or Dictionary, this is the place to start looking. I certainly don't think commenting on all the namespaces is useful, but the most used ones like this or System.Linq seem like they might be worth a comment at the start.